protected/humhub/modules/file/handler/ImageFileHandler.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) 2023 HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+namespace humhub\modules\file\handler;
+
+use Yii;
+
+/**
+ * ImageFileHandler allows uploading images files type
+ * @since 1.15
+ */
+class ImageFileHandler extends BaseFileHandler
+{
+
+    /**
+     * @inheritdoc
+     */
+    public function getLinkAttributes()
+    {
+        return [
+            'label' => Yii::t('MailModule.base', 'Attach an image'),
```

Comment:
Same here, should separate from external modules, so `FileModule` should be used.